.github/codecov.yml

```diff
@@ -1,16 +1,27 @@
 coverage:
   status:
-    patch:
-      default:
-        target: "80"
     project:
       default:
-        target: auto
-        threshold: 1
+        informational: true
+    patch:
+      default:
+        informational: true
 ignore:
-  - "test/"
-  - "benchmark/"
-  - "internal/testutils"
-  - ".*.pb.go"
+  - "benchmark/**/*"
+  - "stress/**/*"
+  # All 'pb.go's
+  - "**/*.pb.go"
+  # Tests and test related files.
+  - "test/**/*"
```

Comment:
`**/test` instead of anything with test?

Similarly:
- `**/testutils`
- `**/testdata`

Reply:
Done